out/namespaces/action.d.ts

```diff
@@ -196,12 +196,12 @@ export namespace Action {
         /**
          * Sets the text color for the badge.
          */
-        setBadgeTextColor(details: SetBadgeTextColorDetailsType): void;
+        setBadgeTextColor(details: SetBadgeTextColorDetailsType): Promise<void>;
 
         /**
          * Gets the text color of the browser action badge.
          */
-        getBadgeTextColor(details: Details): void;
+        getBadgeTextColor(details: Details): Promise<void>;
```

Comment:
issue: Even though this was wrong before, it's not correct yet either. It's supposed to return a Promise of a [browserAction.ColorArray](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/browserAction/ColorArray).

Reply:
Done.